data/news/contra_costa.json

```diff
@@ -3,96 +3,6 @@
-      "title": "Health Order: Updated Mass Quarantine Order",
-      "date_published": "2020-09-14T00:00:00-07:00"
-    },
-    {
-      "id": "https://cchealth.org/press-releases/2020/0914-Health-Order.php",
-      "url": "https://cchealth.org/press-releases/2020/0914-Health-Order.php",
-      "title": "Press Release:\u00a0Contra Costa Updates Health Order to Align with California's COVID-19 Blueprint",
-      "date_published": "2020-09-14T00:00:00-07:00"
-    },
-    {
-      "id": "https://www.coronavirus.cchealth.org/frequently-asked-questions",
-      "url": "https://www.coronavirus.cchealth.org/frequently-asked-questions",
-      "title": "Updated Frequently Asked Questions",
-      "date_published": "2020-09-08T00:00:00-07:00"
-    },
-    {
-      "id": "https://813dcad3-2b07-4f3f-a25e-23c48c566922.filesusr.com/ugd/84606e_05b2f6cd4fff43ea9574506930a8edba.pdf",
-      "url": "https://813dcad3-2b07-4f3f-a25e-23c48c566922.filesusr.com/ugd/84606e_05b2f6cd4fff43ea9574506930a8edba.pdf",
-      "title": "Flyer: Avoid the 3-C's|Evite el 1-2-3",
-      "date_published": "2020-09-08T00:00:00-07:00"
-    },
-    {
-      "id": "https://www.coronavirus.cchealth.org/health-orders",
-      "url": "https://www.coronavirus.cchealth.org/health-orders",
-      "title": "Health Order: Updated Social Distancing Order",
-      "date_published": "2020-09-04T00:00:00-07:00"
-    },
-    {
-      "id": "https://813dcad3-2b07-4f3f-a25e-23c48c566922.filesusr.com/ugd/ee8930_cd269594a79741c69a16a58f649c0dbc.pdf",
-      "url": "https://813dcad3-2b07-4f3f-a25e-23c48c566922.filesusr.com/ugd/ee8930_cd269594a79741c69a16a58f649c0dbc.pdf",
-      "title": "Social Gatherings Safety Checklist I Lista de Requisitos para Prevenci\u00f3n\u00a0para Reuniones Sociales",
-      "date_published": "2020-09-04T00:00:00-07:00"
-    },
-    {
-      "id": "https://813dcad3-2b07-4f3f-a25e-23c48c566922.filesusr.com/ugd/ee8930_6eaa4b83326d4ac8bd93400b25c9775d.pdf",
-      "url": "https://813dcad3-2b07-4f3f-a25e-23c48c566922.filesusr.com/ugd/ee8930_6eaa4b83326d4ac8bd93400b25c9775d.pdf",
-      "title": "Infographic: Timeline Scenarios for In-Person Instruction",
-      "date_published": "2020-09-04T00:00:00-07:00"
-    },
-    {
-      "id": "https://813dcad3-2b07-4f3f-a25e-23c48c566922.filesusr.com/ugd/ee8930_03bf4ba0d1c14d85887fd31ef4242268.pdf",
-      "url": "https://813dcad3-2b07-4f3f-a25e-23c48c566922.filesusr.com/ugd/ee8930_03bf4ba0d1c14d85887fd31ef4242268.pdf",
-      "title": "Flyer & Poster: Do The Right Thing, We Have the Power",
-      "date_published": "2020-09-04T00:00:00-07:00"
-    },
-    {
-      "id": "https://cchealth.org/press-releases/2020/0903-Labor-Day-Weekend.php",
-      "url": "https://cchealth.org/press-releases/2020/0903-Labor-Day-Weekend.php",
-      "title": "Press Release: Health\u00a0Officials Offer Tips for Not Spreading COVID-19 During Labor Day Weekend",
-      "date_published": "2020-09-03T00:00:00-07:00"
-    },
```

Comment:
These removals are a bug being addressed in https://github.com/sfbrigade/data-covid19-sfbayarea/pull/138.

Reply:
…which is now merged. :)